ToDo/src/main/java/com/todo/demo/Service/NoteService.java

```diff
@@ -0,0 +1,24 @@
+package com.todo.demo.Service;
+
+import com.todo.demo.Note.Note;
+import com.todo.demo.Note.Priority;
+
+import java.util.List;
+
+public interface NoteService {
+
+    public String sayHello();
+
+    //Create
+    public void createNote(Note note);
+
+    //Read
```

Comment:
comments are not needed where use is implicit

Reply:
Okay.